x-pack/plugins/session_view/public/components/process_tree_alerts/index.tsx

```diff
@@ -30,11 +30,11 @@ export function ProcessTreeAlerts({
   const styles = useStyles();
 
   useEffect(() => {
-    const jumpToAlert = alerts.find((alert) => alert.kibana?.alert.uuid === jumpToAlertID);
-    if (jumpToAlertID && jumpToAlert) {
+    const jumpToAlert = alerts.find((alert) => alert.kibana?.alert.uuid === investigatedAlertId);
```

Comment:
preempt: `investigatedAlertId && alert.kibana?.alert.uuid === investigatedAlertId` to also check for `undefined === undefined`